tardis/util/base.py

```diff
@@ -10,6 +10,8 @@
 from tardis import constants
 from astropy import units as u
 from pyne import nucname
+from numba import njit
+from tqdm.autonotebook import tqdm
```

Comment:
why do you import this?

Reply:
actually I ran this on jupyter notebook and without this it was giving some warnings.

Reply:
but you do not use tqdm in this function, right?

Reply:
Yes I didn't. I will remove it.